packages/connect-popup/src/view/common.tsx

```diff
@@ -122,9 +122,7 @@ export const initMessageChannel = async (
         ]);
 
     const iframe = getIframeElement();
-    if (!iframe) {
-        throw ERRORS.TypedError('Popup_ConnectionMissing');
-    }
```

Comment:
so this happened in mv2 right? but not in web?. I still wonder why and if this is the correct solution. 

I think that getIframeElement should be maybe at least called again before line 141 maybe?

Reply:
after POPUP.HANDSHAKE (which is sent by iframe if I remember correctly), iframe should of course always exist. Maybe 1 second timeout in promise.race is too little?

Reply:
Yes, the issue is always in webextension mv2 and mv3, it works ok on web.